README.md

```diff
@@ -130,10 +130,11 @@ If you are using [Expo](https://expo.io/), instead of adding the `rn-cli.config.
 
 ### Using TypeScript
 
-If you are using TypeScript, you need to add this to your `declarations.d.ts` file (create one if you don't have one already):
+If you are using TypeScript, you need to add this to your `declarations.d.ts` file (create one if you don't have one already, but don't put in root):
```

Comment:
Could this instead say `, but don't put in the root folder of your project` to be a bit more specific about what needs to be done?

Reply:
True, fixed it